app/move/controllers/create/move-details.js

```diff
@@ -37,12 +37,35 @@ class MoveDetailsController extends CreateBaseController {
         date: filters.formatDateWithDay(res.locals.TOMORROW),
       })
 
+      const itemForPrisonRecall = find(
+        req.form.options.fields.move_type.items,
+        { value: 'prison_recall' }
+      )
+      if (req.session.user.role === 'Prison') {
+        itemForPrisonRecall.conditional = 'move_to_prison_name'
+        delete req.form.options.fields.additional_information
+        const items = await this.populatePrisonAutocomplete()
+        set(req, 'form.options.fields.move_to_prison_name.items', items)
```

Comment:
It might be worth taking a look at the previous logic for this:
https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/commit/c49713a0e582ddbaf17d4ea925855f650209f068#diff-14d99f63615581969e8020b943229313

https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/commit/bbeb37183ac66517c2725dc1ee97802f9d73894a#diff-4e531508b9cdbccfd41cd6b15225d987

I think currently it won't store the value of the prison as `from_location` which we'll need when sending to the API.

Reply:
Now it sends the correct data:
```
{
   "from_location": "e517a984-bb11-4738-aa93-229c22c2d459",
   "to_location": "240f8ea3-f3cf-4598-90b6-4c86c90d6dfc",
   "move_type": "prison_recall",
   "to_location_prison_recall": "240f8ea3-f3cf-4598-90b6-4c86c90d6dfc",
   "to_location_court_appearance": "",
   "date": "2020-02-21",
   "date_type": "today",
   "date_custom": "",
   "can_upload_documents": false
}
```